core/src/main/resources/jenkins/slaves/systemInfo/ClassLoaderStatisticsSlaveInfo/systemInfo.groovy

```diff
@@ -1,20 +1,21 @@
 import hudson.slaves.SlaveComputer
 
 def fmt = new java.text.DecimalFormat("0.0")
-def right = 'text-align: right'
 if (my instanceof SlaveComputer) {
     SlaveComputer c = my
 
-    table(class: 'bigtable') {
-        tr {
-            th _('Loading Type')
-            th _('Time (s)')
-            th _('Count')
+    table(class: 'jenkins-table') {
```

Comment:
`width: auto` should make the table compact

Reply:
I wasn't able to find a way to apply `width: auto` in that change and have any different result than is shown in the screenshot.

I also tried [`table-layout: auto`](https://www.w3schools.com/cssref/pr_tab_table-layout.asp) without success

Reply:
In my plugin I do
`<table class="jenkins-table sortable" style="width: auto">`
which works fine.
In the old styles there was `bigtable--compact` seems this is missing

Reply:
Thanks for the pointer.  Much appreciated.

I was able to make it work with `style: "width: auto"` but I prefer the layout without the `width: auto`.  The `width: auto` makes the separation between column headings less clear to me.  Open to other opinions, but I prefer it full width.  I'll leave it with the current layout unless there is strong feedback that it needs more styling.

Reply:
I'd rather not do that, as that breaks with the styling of existing tables.

Reply:
> I'd rather not do that, as that breaks with the styling of existing tables.

When you say, "I'd rather not do that", I assume you mean "I would rather we not add `width: auto` to the table".  Did I understand correctly @NotMyFault ?